test-app/src/main/java/org/readium/r2/testapp/reader/AudioReaderFragment.kt

```diff
@@ -1,53 +1,53 @@
 package org.readium.r2.testapp.reader
 
+import android.annotation.SuppressLint
 import android.media.AudioManager
 import android.os.Bundle
 import android.text.format.DateUtils
 import android.view.LayoutInflater
+import android.view.MotionEvent
 import android.view.View
 import android.view.ViewGroup
 import android.widget.SeekBar
 import androidx.fragment.app.activityViewModels
-import androidx.lifecycle.asLiveData
 import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
 import org.readium.r2.navigator.ExperimentalAudiobook
-import org.readium.r2.navigator.MediaNavigator
-import org.readium.r2.navigator.Navigator
-import org.readium.r2.navigator.media.MediaService
+import org.readium.r2.navigator.media2.MediaNavigator
+import org.readium.r2.shared.publication.Link
+import org.readium.r2.shared.publication.Locator
 import org.readium.r2.shared.publication.services.cover
 import org.readium.r2.testapp.R
+import org.readium.r2.testapp.R2App
 import org.readium.r2.testapp.databinding.FragmentAudiobookBinding
+import timber.log.Timber
+import kotlin.math.roundToLong
 import kotlin.time.Duration
 import kotlin.time.Duration.Companion.seconds
 import kotlin.time.DurationUnit
 import kotlin.time.ExperimentalTime
 
-@OptIn(ExperimentalAudiobook::class, ExperimentalTime::class)
-class AudioReaderFragment : BaseReaderFragment() {
```

Comment:
I think we can keep it named `AudioReaderFragment` as we won't keep the media1 code in the test app.